pkg/cloud/services/network/routetables.go

```diff
@@ -362,6 +362,13 @@ func (s *Service) getNatGatewayPrivateRoute(natGatewayID string) *ec2.CreateRout
 	}
 }
 
+func (s *Service) getNat64PrivateRoute(natGatewayID string) *ec2.CreateRouteInput {
```

Comment:
Question: I don't know what makes this a private route. I see `getNatGatewayPrivateRoute()` also implemented similarly.

Reply:
Oh, the "private" here means that the route is added to the route table associated with a private subnet.

Now that I think about it more, this route should be added for both private and public subnets (i.e. public subnet only install) to allow IPv6-only workloads in those subnets to reach IPv4-only internet services.